flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -567,6 +600,17 @@ protected boolean isLate(W window) {
 	}
 
 	/**
+	 * Decide if a record is currently late, based on current watermark and allowed lateness.
+	 *
+	 * @param element The element to check
+	 * @return The element for which should be considered when sideoutputs
+	 */
+	protected boolean isLate(StreamRecord<IN> element){
```

Comment:
I must have removed the check by accident. I think we agreed to rename this to something more meaningful and keep it, right?